contents/tutorials/calculating-new-returning-users.md

```diff
@@ -0,0 +1,93 @@
+
+> **How does Google Analytics calculate new vs returning users?** When you first visit a site, Google Analytics generates a client ID for your device and sets it in your cookies. GA uses this client ID cookie to calculate if a user is new (didn’t have cookie set) or returning (had cookie set). This, like any tracking, isn’t 100% accurate.
+
+## Calculating new users
+
+There are multiple ways to calculate new users in PostHog depending on your situation and need.
+
+### Completed event for the first time cohort
+
+The first and easiest is creating a cohort where the users completed an event (such as pageviews, identify, custom sign up) for the first time. To set this up, go to the cohort tab, click new cohort, enter a title and description, then choose "completed an event for the first time" with your event and recency. 
+
+![New users](../images/tutorials/calculating-new-returning-users//new.png)
+
+You can then use this cohort of new users for further analysis.
+
+> **Note:** since this is a behavioral cohort (based on events), this cohort won’t work with feature flags.
+
+### First time event tracker
+
+You can set up and use the [first time event tracker app](/docs/apps/first-time-event-tracker) to add a property on users to say when they completed the pageview, identify, or custom event for the first time.
+
+To set it up, search for "First Time Event Tracker" in your PostHog instance, set the event you want to track in the configuration (like `$pageview`) and enable the app. 
+
+![First time event tracker](../images/tutorials/calculating-new-returning-users/first.png)
+
+You can then filter for events where `is_first_event_user` is `true` recently to get new users for analysis.
+
+### Manually set user property
+
+If neither of these works for you, you can set up a user property manually in a custom event. 
+
+For example, when a user first uses your API, set their creation date in their user properties. To do this, you can capture an event at the same time, and use the `$set_once` property like this:
+
+```python
+posthog.capture(
+    '<ph_project_api_key>',
+    event='new user created',
+    properties={ '$set_once': { 'created_at': '2023-04-24T22:02:02' } }
+)
+```
+
+This is especially useful for projects with many custom events, on the backend, not using autocapture, or not using `posthog-js`. 
+
+## Calculating returning users
+
+Unlike new users, there is one way to calculate returning users. It is identifying users who aren’t new but completed an event recently. To create a cohort for this, you can use the new user cohort as an anti-cohort (choose "Not in cohort"), and include another criteria for completing an event recently.
+
+![Returning users](../images/tutorials/calculating-new-returning-users/returning.png)
+
+If you created a `created_at` user property, you can also filter for users who weren’t created recently but completed the event. 
```

Comment:
```suggestion
Alternatively, if you created a `created_at` user property, you can also filter for users who weren’t created recently but completed the event. 
```

(also, this seems to apply that are two ways and not one way?)

Reply:
Good point